### PR TITLE
[MM-44931] Allow zoom in/out when Shift is pressed

### PR DESCRIPTION
**Pull request by tboulis**
#### Summary
Add hidden MenuItems that add keyboard shortcuts for Zoom In/Out when Shift is pressed.
I tried using the default role for `zoomIn` and `zoomOut` that electron provides, but it didn't work on Windows, so I followed this approach.

#### Ticket Link
  Fixes https://mattermost.atlassian.net/browse/MM-44931

#### Checklist

- [x] Added or updated e2e tests
- [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
- [x] completed [Mattermost Contributor Agreement](https://mattermost.com/contribute/)
- [x] executed `npm run lint:js` for proper code formatting

#### Device Information
This PR was tested on:
- macbook pro, Macos Monterey 12.4 
- pc, Windows 11
- pc, Ubuntu 22

#### Release Note
```release-note
Fix zoom in/out not working when Shift is pressed.
```

### CODE REVIEW

File: e2e/specs/menu_bar/view_menu.test.js [hunk 1]
```diff
@@ -126,6 +138,28 @@ describe('menu/view', function desc() {
         zoomLevel.should.be.greaterThan(1);
     });
 
+    it('MM-44931 Zoom in when CmdOrCtrl+Shift+Plus is pressed', async () => {
```
Comment (devinbinnie) [inline e2e/specs/menu_bar/view_menu.test.js:141]: Thanks for adding these tests!
For better organization, I'd like to add this as part of the MM-T818 test case, which is in the same file. So what we can do here is put them both under a `describe()` with the label `MM-T818` and add this one as part 2. Have a look at the other test files and you can see how we split up the pieces.

**Review by devinbinnie — CHANGES_REQUESTED** [review]
Thanks for the PR @tboulis!
Everything looks great and works well, I just had a couple requests for better code organization.

File: e2e/specs/menu_bar/view_menu.test.js [hunk 2]
```diff
@@ -110,36 +122,84 @@ describe('menu/view', function desc() {
...
-        await asyncSleep(1000);
-        const zoomLevel = await browserWindow.evaluate((window, id) => window.getBrowserViews().find((view) => view.webContents.id === id).webContents.getZoomFactor(), firstServerId);
-        zoomLevel.should.be.greaterThan(1);
+            robot.keyTap('=', [env.cmdOrCtrl]);
+            await asyncSleep(1000);
+            const zoomLevel = await browserWindow.evaluate((window, id) => window.getBrowserViews().find((view) => view.webContents.id === id).webContents.getZoomFactor(), firstServerId);
+            zoomLevel.should.be.greaterThan(1);
+        });
+
+        it('MM-44931 Zoom in when CmdOrCtrl+Shift+Plus is pressed', async () => {
```
Comment (devinbinnie) [inline e2e/specs/menu_bar/view_menu.test.js:142]: All looks good, I just wanted to name the test cases for the test cases we have stores in JIRA.

```suggestion
        it('MM-T818_1 Zoom in when CmdOrCtrl+Shift+Plus is pressed', async () => {
```

File: e2e/specs/menu_bar/view_menu.test.js [hunk 3]
```diff
@@ -110,36 +122,84 @@ describe('menu/view', function desc() {
...
-        await asyncSleep(1000);
-        const zoomLevel = await browserWindow.evaluate((window, id) => window.getBrowserViews().find((view) => view.webContents.id === id).webContents.getZoomFactor(), firstServerId);
-        zoomLevel.should.be.lessThan(1);
+            robot.keyTap('-', [env.cmdOrCtrl]);
+            await asyncSleep(1000);
+            const zoomLevel = await browserWindow.evaluate((window, id) => window.getBrowserViews().find((view) => view.webContents.id === id).webContents.getZoomFactor(), firstServerId);
+            zoomLevel.should.be.lessThan(1);
+        });
+
+        it('MM-44931 Zoom out when CmdOrCtrl+Shift+Minus is pressed', async () => {
```
Comment (devinbinnie) [inline e2e/specs/menu_bar/view_menu.test.js:182]:
```suggestion
        it('MM-T819_1 Zoom out when CmdOrCtrl+Shift+Minus is pressed', async () => {
```

**Review by devinbinnie — CHANGES_REQUESTED** [review]
One last little thing

**Review by devinbinnie — APPROVED** [review]
Thanks @tboulis! Looks good now!

**Review by hmhealey — APPROVED** [review]
Thanks for the PR! I wouldn't have thought you could just hide a menu item (vs removing it entirely), but that works for me

**Pull request merged**